sdk/python/kfp/dsl/for_loop.py

```diff
@@ -149,20 +173,16 @@ def __getattr__(self, name: str):
                 subvar_name=name,
             ))
 
-    def _make_name(self, code: str):
-        """Makes a name for this loop argument from a unique code."""
-        return f'{self.LOOP_ITEM_PARAM_NAME_BASE}-{code}'
-
     @classmethod
     def from_pipeline_channel(
         cls,
-        channel: pipeline_channel.PipelineParameterChannel,
-    ) -> 'LoopArgument':
-        """Creates a LoopArgument object from a PipelineParameterChannel
+        channel: pipeline_channel.PipelineChannel,
```

Comment:
Should this still be a `PipelineParameterChannel`?

Reply:
Done.